markdowns/mentee_checksheet.md

```diff
@@ -1,24 +1,15 @@
 # Checksheet for Mentees Working On A Project
 
- ```Below is a high-level checksheet for your project, from start to finish. This is not meant to replace the steps in "Getting Started". It is to help answer questions over the overall project progression, which students seem to often run into.```
-
-<input type="checkbox"/>  Fork the Sample Project, and have your mentor link it to the amp-lab.org website
-
-<input type="checkbox"/>  First Meeting with Mentor
-
-<input type="checkbox"/>  Research your project, and reach out to mentor with any questions you may have
-
-<input type="checkbox"/> Submit your BOM (If you need parts) to the Discord channel **purchase-requests** so that we can order your parts. __Note: Mentors will verify that the parts you are requesting are reasonable. You may be asked to revise your project's BOM__
-
-<input type="checkbox"/>  Await parts, and go pick them up when instructed
-
-<input type="checkbox"/>  Build your first draft. If you need more parts, feel free to submit another BOM
-
-<input type="checkbox"/>  Complete your first draft. This is called "MVP", or "Minimal Viable Product" it should be a good first-attempt with base-bones functionality. This is the version the Amp Lab will complete. **Note: With the lab, you will always make two versions of your project: a basic one for the lab to keep, and a better draft that you can keep**
-
-<input type="checkbox"/>  Make the second draft, or the one you will be keeping
-
-<input type="checkbox"/>  Update documentation
-
-<input type="checkbox"/>  Project Complete. Well done! Have a cookie :)
-
+> Below is a high-level checksheet for your project, from start to finish. This is not meant to replace the steps in "Getting Started". It is to help answer questions over the overall project progression, which students seem to often run into.
+
```

Comment:
This doesn't work

Reply:
the checkboxes dont work or the ">" doesnt work?

Reply:
![Screenshot from 2023-08-30 00-17-59](https://github.com/Amp-Lab-at-VT/website/assets/33364183/bb0c47de-3e0c-46d9-af87-752abfb70b8e)

Reply:
See how the checkboxes have disappeared? This is what I meant. Sorry it took me literal ages to get to reviewing this

Reply:
ohhh I see. that makes sense. its confusing because vscode by default does not support `[ ]` as checkboxes, github does, but then the markdown plugin does not. 
Ill revert it back in a bit